content/applications/sales/sales/products_prices/products/variants.rst

```diff
@@ -105,12 +77,19 @@ The :guilabel:`Display Type` options are:
 - :guilabel:`Select`: options appear in a drop-down menu on the product page of the online store.
 - :guilabel:`Color`: options appear as small, colored squares, which reflect any HTML color codes
   set, on the product page of the online store.
+- :guilabel:`Multi-checkbox (option)`: options appear as selectable checkboxes on the product page
+  of the online store.
 
 .. image:: variants/display-types.png
    :align: center
    :alt: Display Types on Product Configurator on the online store in Odoo.
 
-The :guilabel:`Variants Creation Mode` field informs Odoo when to automatically create a new variant
+.. note::
```

Comment:
I feel like the definition and the "note" part could be flipped? Also it looks like the `Variant Creation Mode`is only in a note- could be expanded to have a small sentence or two outside of the note? Just an idea- not necessary.